AdvLoggerPkg/Include/Guid/AdvancedLoggerPreDxeLogs.h

```diff
@@ -0,0 +1,31 @@
+/** @file
+  This file defines GUIDs and data structure for producing HOB entry describing
+  the location of pre-DXE logs. For platforms which use a Pre-DXE AdvancedLoggerLib
+  implementation, this HOB is not necessary.
+
+  Copyright (C) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#ifndef __ADVANCED_LOGGER_PRE_DXE_LOGS_H__
+#define __ADVANCED_LOGGER_PRE_DXE_LOGS_H__
+
+// The region specified by this HOB entry should contain raw log output.
+// The Advanced Logger library will serialize the contents of the buffer
+// when adding it to the advanced logger output.
+typedef struct _ADVANCED_LOGGER_PRE_DXE_LOGS_HOB {

```

Comment:
I think it'd be a good idea to pack the structure given separately compiled environments might be producing it.

Reply:
Updated